app/models/concerns/ats/ashbyhq/application_fields.rb

```diff
@@ -4,18 +4,18 @@ module ApplicationFields
       include FaradayHelpers
 
       def get_application_question_set(job, _data)
-        p "Getting AshbyHQ application criteria"
-        data = fetch_job_api_data(job.ats_job_id, job.company.ats_identifier)
-        return [] unless (job_data = data&.dig('data', 'jobPosting'))
+        data = fetch_job_api_data(job)
+        return [] unless (data = data&.dig('data', 'jobPosting'))
 
-        job.update(deadline: job_data['applicationDeadline']) # not sure what format
-        Importer::GetAshbyFields.call(job, job_data)
+        job.update(deadline: data['applicationDeadline'])
```

Comment:
Think we can standardise this a bit further between ATS systems - to discuss

Reply:
I don't like that part but did not have enough knowledge to decide what to do with.
to be discussed with Dan.

Reply:
@daniel-sussman - one to discuss